google/cloud/internal/opentelemetry.h

```diff
@@ -94,6 +95,19 @@ opentelemetry::nostd::shared_ptr<opentelemetry::trace::Span> MakeSpan(
     opentelemetry::trace::StartSpanOptions const& options =
         DefaultStartSpanOptions());
 
+/**
+ * Start a span with a @p name and @p attributes using templates.
```

Comment:
I am not use "using templates" adds a lot.

Reply:
ack, removed.